frontend/Views/Widgets/ViewGraphedStats/AssigneeCell.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+    import { getUser, getPicture } from "../../../Common/UserUtils";
+    import { userList } from "../../../Stores/UserlistSubscriber";
+    import type { TestRun } from "./Interfaces";
+
+    export let run: TestRun;
+
+    let users = {};
+    $: users = $userList;
+
+    $: assignee = run.assignee;
+    $: user = assignee && assignee !== "Unassigned" ? getUser(assignee, users) : null;
+    $: pictureUrl = user ? getPicture(user.picture_id) : "";
+    $: fullName = user ? (user.full_name || user.username || assignee) : "";
```

Comment:
All variables in this block are undefined and this file will not compile. Also, the reactivity directive is not needed.